nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/log/LogLevelCellFactory.java

```diff
@@ -0,0 +1,45 @@
+package org.nmrfx.processor.gui.log;
+
+import de.jensd.fx.glyphs.fontawesome.FontAwesomeIcon;
+import javafx.scene.control.TableCell;
+import javafx.scene.paint.Color;
+import org.controlsfx.glyphfont.Glyph;
+
+import ch.qos.logback.classic.Level;
```

Comment:
I had hoped to abstract the dependency to logback. It isn't a good news to see this import in UI code. 
Is there any way to use the `org.nmrfx.processor.gui.log.LogLevel` enum here?

If it is only for the `>=` checks, we could use `LogLevel.ordinal()` instead.

Reply:
Was able to use LogLevel.ordinal instead, removed dependency to logback